app/misc/graphing.ts

```diff
@@ -10,15 +10,36 @@ let commitHistory = [];
 let commitList = [];
 let spacingY = 100;
 let spacingX = 80;
+let tagSpacingY = 100;
+let tagSpacingX = 300;
 let parentCount = {};
 let columns: boolean[] = [];
 let edgeDic = {};
 let numOfCommits = 0;
 let branchIds = {};
+let tagIds = {};
+let unumberPrev = 0;
```

Comment:
This is the previous unique number, correct?

Reply:
@colinwerner this is the smallest previous unique number.

Reply:
@SiRumCz the smallest? But it gets updated when date > unumberPrev?